app/models/dish.rb

```diff
@@ -1,4 +1,14 @@
 class Dish < ApplicationRecord
   validates_presence_of :name, :description
   belongs_to :chef
+  has_many :dish_ingredients
+  has_many :ingredients, through: :dish_ingredients
+
+  def calorie_count
+    calories = 0
```

Comment:
Whenever you are doing Ruby .map, .each or other enumerable in a model method on an IC, consider -- is there an AR method that could help me do this more easily?

in this case: `ingredients.sum(:calorie_count)`